### PR TITLE
Run tests on PyPy, PyPy3, fixes #94

### LINKED ISSUE #94: add pypy, pypy3 support

### PR DESCRIPTION
**Pull request by mmaslowskicc**

### CODE REVIEW

File: src/pytest_dbfixtures/factories/postgresql.py [hunk 1]
```diff
@@ -233,7 +233,10 @@ def postgresql_factory(request):
         """
         proc_fixture = request.getfuncargvalue(process_fixture_name)
 
-        psycopg2, config = try_import('psycopg2', request)
+        if platform.python_implementation() == 'PyPy':
```
Comment (mmaslowskicc) [inline src/pytest_dbfixtures/factories/postgresql.py]: I don't like this, while I'm not sure if there is a more typical way to run an available package among several supported ones.

**Comment by pwilczynskiclearcode** [comment]
:+1:

File: src/pytest_dbfixtures/factories/postgresql.py (hunk 1 above)
Comment (fizyk) [inline src/pytest_dbfixtures/factories/postgresql.py]: ideally, it would be kept in compat.py for these ifs

Comment (fizyk) [inline src/pytest_dbfixtures/factories/postgresql.py]: or maybe some modification to try_import?

**Pull request closed without merging**